revelation/ember-cli-revelation-ui
# Adds default placeholder avatar image

### What does this PR do?

Adds a default placeholder avatar to the avatar component.  Had to slightly refactor the code, as the placeholder image needed to work as a background image.

### How should this be tested?

- npm link to this branch
- checkout if not merged yet [THIS BRANCH](https://invent.focusvision.com/Portland/revelation-frontend-admin/tree/feature/ks/fix-broken-avatar)
- Delete your avatar.
```
u=User.find(:id)
u.avatar.destroy
```
- Navigate to digest
- Ensure placeholder image is visible
- Re-add an avatar
- Make sure avatar is present

### Questions?

### Related Issues

Related to #175

- [x] QA Complete
